mopro-ffi/build.rs

```diff
@@ -1,6 +1,14 @@
 fn main() {
     #[cfg(feature = "circom")]
-    if std::env::var("MOPRO_FFI_LINK_TEST_WITNESS").unwrap_or_default() != "" {
-        rust_witness::transpile::transpile_wasm("../test-vectors/circom".to_string());
+    {
+        #[cfg(feature = "circom-wit-rustwitness")]
+        if std::env::var("MOPRO_FFI_LINK_TEST_WITNESS").unwrap_or_default() != "" {
+            rust_witness::transpile::transpile_wasm("../test-vectors/circom".to_string());
+        }
+
+        #[cfg(feature = "circom-wit-witnesscalc")]
+        {
+            witnesscalc_adapter::build_and_link("../test-vectors/circom/witnesscalc");
```

Comment:
you can just put in `circom`
and I think it stills needs the 
```rs
if std::env::var("MOPRO_FFI_LINK_TEST_WITNESS").unwrap_or_default() != "" {
```
to prevent from building witnesscalc if using in `test-e2e` or `mopro-example-app`

Reply:
fixed in [35305a1](https://github.com/zkmopro/mopro/pull/343/commits/35305a125cf40fff17b1e3c2d640ce5032c79f8d)